lib/rucio/daemons/conveyor/common.py

```diff
@@ -164,7 +172,21 @@ def bulk_group_transfer(transfers, policy='rule', group_bulk=200, fts_source_str
             if 'adler32' in list(file['metadata'].keys()) and file['metadata']['adler32']:
                 file['checksum'] = 'ADLER32:%s' % str(file['metadata']['adler32'])
 
-        job_params = {'verify_checksum': verify_checksum,
+        external_host = transfer['external_host']
+        scope = file['metadata']['scope']
+        activity = file['activity']
+
+        if external_host not in grouped_transfers:
+            grouped_transfers[external_host] = {}
+            if USER_TRANSFERS not in ['cms'] or activity not in USER_ACTIVITY:
+                grouped_jobs[external_host] = []
+            elif activity in USER_ACTIVITY:
+                grouped_jobs[external_host] = {}
+                if scope not in grouped_transfers[external_host]:
+                    grouped_transfers[external_host][scope] = {}
+                    grouped_jobs[external_host][scope] = []
+
+        job_params = {'verify_checksum': True if file['checksum'] and file['metadata'].get('verify_checksum', True) else False,
```

Comment:
you should just use the verify_checksum that was created a few lines above here. otherwise you'll nullify the `source`/`destination`/`both` checksum capability

Reply:
right, changing it.